src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -5,7 +5,6 @@
     <AssemblyName>FakeItEasy</AssemblyName>
     <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
I am surprised there's no equivalent to this in FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj. Maybe the explicit mention of the .pdb takes its place?

Reply:
Yes

Reply:
yes